src/cobalt.rs

```diff
@@ -262,11 +268,29 @@ pub fn build(config: &Config) -> Result<()> {
                 }
             }
             let src_file = source.join(file_path.as_path());
-            let dest_file = dest.join(file_path);
 
-            fs::copy(src_file.as_path(), dest_file.as_path())
-                .map_err(|e| format!("Could not copy {:?} into {:?}: {}", src_file, dest_file, e))?;
-            debug!("Copied {:?} to {:?}", src_file, dest_file);
+            if file_path.extension().unwrap_or_else(|| OsStr::new("")) == "scss" {
```

Comment:
Could you do `if file_path.extension() == Some("scss")`?

Reply:
Good point. Will do.